### ISSUE TITLE
No nodes after installing

### ISSUE THREAD

**Issue by lucwuyts** [Foddy/node-red-contrib-huemagic#4]
Hi, i just installed version 1.1.1 but still don't see the new nodes.
The log shows the same errors as in my previous post:

19 Sep 14:32:25 - [warn] ------------------------------------------------------
19 Sep 14:32:25 - [warn] [rpi-gpio] Info : Ignoring Raspberry Pi specific node
19 Sep 14:32:25 - [warn] [tail] Not currently supported on Windows.
19 Sep 14:32:25 - [warn] [hue-bridge] SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
19 Sep 14:32:25 - [warn] [hue-light] SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
19 Sep 14:32:25 - [warn] [hue-group] SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
19 Sep 14:32:25 - [warn] [hue-scene] SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
19 Sep 14:32:25 - [warn] [hue-motion] SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
19 Sep 14:32:25 - [warn] [hue-temperature] SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
19 Sep 14:32:25 - [warn] [hue-brightness] SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
19 Sep 14:32:25 - [warn] ------------------------------------------------------

**Comment by Foddy** [comment]
Okay.. I might have misplaced the "use strict" command. Can you please try it again with v1.1.2? Sorry…

**Comment by heydemar** [comment]
Thanks for the Update, now i have the Battery-Status at my second Motion-Sensor :)

**Comment by lucwuyts** [comment]
I have to disappoint you.  Still the same problem here.
Do i have to change a setting for this "strict mode" ?

**Comment by lucwuyts** [comment]
I solved it...
I first uninstalled with the command npm un ....
The output showed the package was removed, but in the node-red pallette manager there was still version 1.1.0 shown.

Removing it with package manager was not possible (possibly a rights problem??)
So i manualy deleted the package from   user-folder\node-modules
And the nodes showed up…
Package manager did also show 1.1.2 although this was uninstalled before.

To be sure i removed this package with package manger, and reinstalled from within package manager.

As soon as i find the time, i start playing with it.

Thanks for this package!!
